sysbrokers/IB/config/ib_config_futures.csv

```diff
@@ -188,7 +193,7 @@ EUROSTX200-LARGE,DJ200L,EUREX,EUR,50,1,FALSE
 FANG,NYFANG,NYBOT,USD,5,1,FALSE
 FED,ZQ,CBOT,USD,4167,1,FALSE
 FEEDCOW,GF,CME,USD,50000,100,FALSE
-FTSE100,Z,ICEEU,GBP,10,1,FALSE
+FTSE100,Z,ICEEU,GBP,1000,100,FALSE
```

Comment:
I'm not checking most of these multiplier changes, as I don't trade the product.

Reply:
![image](https://github.com/robcarver17/pysystemtrade/assets/13718675/4cd16c07-8656-4d43-82e3-91259e207e6b)
In this case, the pointsize is 10 while the IB multiplier is 1000. Therefore, the IB magnifier needs to be 100, if I understand correctly.